### ISSUE TITLE
[Suggestion] Material Blocks

### ISSUE THREAD

**Issue by Durpady** [Benimatic/twilightforest#29]
You've added a Knightmetal Block.  What about blocks for Ironwood, Fiery Ingots and (maybe, could be composed of 4 instead of 9) Steeleaf?

**Comment by Ephys** [comment]
Agreed that would be awesome even if they're purely decorative (fiery could burn anyone walking on it without fiery or fire protection boots ?)

**Comment by Benimatic** [comment]
I may add these sometime, as this seems to be a popular suggestion.
